src/delivery/command/init.rs

```diff
@@ -286,6 +287,22 @@ fn verify_config_get_build_cookbook_path<P>(p_path: P) -> DeliveryResult<Option<
     Ok(Some(PathBuf::from(".delivery/build_cookbook")))
 }
 
+// Create or update the delivery git remote
+pub fn create_or_update_git_remote(config: &Config) -> DeliveryResult<()> {
+    sayln("cyan", "Setting up the 'delivery' git remote...");
+    if project::verify_git_remote(config)? {
```

Comment:
`verify_git_remote` returns (simplified) `Ok(expected =! actual)`, so this is a bit confusing. If it was called `git_remote_needs_update` this would be one indirection less...

Alternatively, if it it returned `Ok(expected == actual)`, and you'd check `if !project:verify_git_remote()`, this would also be a little less confusing in my opinion ;)